bin/config

```diff
@@ -111,20 +131,27 @@ check_opts() {
     done
 
     if [ "$found" == false ]; then
-      echo "Invalid function name"
-      see_help_dev
+      echo "Invalid Gorup Option"
```

Comment:
Group?

Reply:
Meant it to be `Function group` 

Perhaps 'Options' may better suited?

Reply:
Just noticed typo gorup vs group ... its fine.